static/app/components/events/breadcrumbs/combinedBreadcrumbsAndLogsSection.tsx

```diff
@@ -0,0 +1,54 @@
+import {Fragment} from 'react';
+
+import type {BreadcrumbsDataSectionProps} from 'sentry/components/events/breadcrumbs/breadcrumbsDataSection';
+import BreadcrumbsDataSection from 'sentry/components/events/breadcrumbs/breadcrumbsDataSection';
+import useOrganization from 'sentry/utils/useOrganization';
+import {LogsPageParamsProvider} from 'sentry/views/explore/contexts/logs/logsPageParams';
+import {LogsIssuesSection} from 'sentry/views/explore/logs/logsIssuesSection';
+import {useExploreLogsTable} from 'sentry/views/explore/logs/useLogsQuery';
+
+/**
+ * This component is a coordinator and provider wrapper to determine which section to display, collapse etc. since only one section should be displayed by default.
+ */
+export function CombinedBreadcrumbsAndLogsSection({
+  event,
+  group,
+  project,
+}: BreadcrumbsDataSectionProps) {
+  const organization = useOrganization();
+  const feature = organization.features.includes('ourlogs-enabled');
+  if (!feature) {
+    // If we don't have the feature, we should skip the providers as they make api calls.
+    return <BreadcrumbsDataSection event={event} group={group} project={project} />;
+  }
+
+  return (
+    <LogsPageParamsProvider isIssuesDetailView traceId={event.contexts?.trace?.trace_id}>
+      <CombinedBreadcrumbsAndLogsSectionContent
+        event={event}
+        group={group}
+        project={project}
+      />
+    </LogsPageParamsProvider>
+  );
+}
+
+function CombinedBreadcrumbsAndLogsSectionContent({
+  event,
+  group,
+  project,
+}: BreadcrumbsDataSectionProps) {
+  const tableData = useExploreLogsTable({});
+  const shouldCollapseLogs = tableData.data.length === 0;
+  return (
+    <Fragment>
+      <BreadcrumbsDataSection
+        event={event}
+        group={group}
+        project={project}
+        initialCollapse={!shouldCollapseLogs}
```

Comment:
@scttcper when I was testing it, this initialCollapse doesn't actually work since it's almost always set by local storage I think, do we have another way of doing "ignore the local storage for initial collapse state"? FoldSection can have another bool for initialCollapse to be fully controlled vs. local storage set.

Reply:
mostly this is a collapsed until expanded for the first time. We assume if you've expanded it, you want to see it expanded the next time you visit.

Reply:
Alright, that's fine with me, I just thought it would be odd to have the logs section highlighted since there are many cases where logs don't exist for traces (since logs have very low usage currently in the product).